packages/utils/src/route.js

```diff
@@ -56,7 +56,11 @@ function cleanChildrenRoutes (routes, isChild = false, routeNameSplitter = '-',
     route.path = isChild ? route.path.replace('/', '') : route.path
     if (route.path.includes('?')) {
       if (route.name.endsWith(`${routeNameSplitter}index`)) {
-        route.path = route.path.replace(/\?$/, '')
```

Comment:
Maybe can be simplified with something like this? (not tested)

```js
route.path = route.path.replace(/\?\/?$/, trailingSlash ? '/' : '')

Reply:
Yes, it can be.
I just do that because of more readability

Reply:
I think it will cover are 4 possible cases (with/without trailingSlash flag x path containing trailing/not) so would be safer too :)